saleor/product/models.py

```diff
@@ -148,7 +153,7 @@ def bulk_create(self, objs, batch_size=None, ignore_conflicts=False):
             objs, batch_size=batch_size, ignore_conflicts=ignore_conflicts
         )
 
-    def collection_sorted(self, user):
+    def collection_sorted(self, user: "User"):
```

Comment:
Have you considered using `__future__.annotations`? https://www.python.org/dev/peps/pep-0563/
```suggestion
    def collection_sorted(self, user: User):
```

Reply:
Did not consider that. I want to keep mostly known style, if that will be new standard then removing all `"` will be done by few clever regexp I think.